omero/users/cli/sessions.rst

```diff
@@ -109,7 +111,7 @@ Session keys can then be reused to switch between stored sessions using the
     $ bin/omero sessions login -k 22fccb8b-d04c-49ec-9d52-116a163728ca
     Server: [localhost]
     Joined session 1f800a16-1dc2-407a-8a85-fb44005306be (root@localhost:4064).
-    $ bin/omero sessions list
+    $ bin/omero sessions who
```

Comment:
This should still be list. A whole section on `sessions who` could either go here or on the sysadmin page.

Reply:
missed that one